db/seeds.rb

```diff
@@ -37,9 +37,8 @@
   section: topic,
   type: GeneralContent,
   state: :published
-}) do |node|
-  node.revise!(content_body: 'lorem ipsum').apply!
-end
+})
+node2.revise!(content_body: 'lorem ipsum').apply!
```

Comment:
I think the error was the wrong node (`node` instead of `node2`) but it's a good idea to keep it in the block because if node2 already exists I guess we won't want to revise it? (Block on `find_or_create_by` only runs on create...)